src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-embed.js

```diff
@@ -27,8 +27,15 @@ export default class EzBtnEmbed extends EzEmbedDiscoverContentButton {
      */
     addEmbed(items) {
         const contentInfo = items[0].ContentInfo.Content._id;
+        let scrollY;
 
-        this.execCommand();
+        if (navigator.userAgent.indexOf('Chrome') > -1) {
+            scrollY = window.pageYOffset;
```

Comment:
`const scrollY = window.pageYOffset;` and the `let` is not needed (applies to all cases).